entries/life-gb/game.json

```diff
@@ -0,0 +1,23 @@
+{
+    "developer": "xoxo_grease",
+    "files": [
+        {
+            "default": true,
+            "filename": "life.gb",
+            "playable": true
+        }
+    ],
+    "license": "MIT",
+    "platform": "GB",
+    "repository": "https://github.com/Brehana/LIFE_gb",
+    "screenshots": [
+        "splash.png"
+    ],
+    "slug": "life-gb",
+    "tags": [
+        "Open Source",
+        "Cellular Automata"
```

Comment:
maybe "Simulation" fits better and it's a tag we currently have in the schema?

Reply:
oops I misinterpreted the "tags" tag. Fixed!